capa/v28.0.1/release.diff

```diff
@@ -0,0 +1,149 @@
+                                                                >  # Generated with:
+                                                                >  # devctl release create --name 28.0.1 --provider capa --base 28
+apiVersion: release.giantswarm.io/v1alpha1                         apiVersion: release.giantswarm.io/v1alpha1
+kind: Release                                                      kind: Release
+metadata:                                                          metadata:
+  name: aws-28.0.0                                              |    creationTimestamp: null
+                                                                >    name: v28.0.1
```

Comment:
`name: v28.0.1` seems wrong. All the others are named `aws-X.Y.Z`. @giantswarm/team-turtles we somehow need to ensure that releases are prefixed correctly, right?

Also, the diff isn't readable in any way because the keys aren't sorted the same.

I'm also concerned that `release.diff` even gets updated if teams now merge various changes into one PR – at best, these files would be checked by CI, but I guess the problem is that you modify a release in so many ways (`devctl release create --some-parameters`, modify `release.yaml` manually, version requirements file?, ...). That's out of scope in this PR, of course.

Reply:
Ah sure. I noticed the naming convention for other releases and [changed it for the release itself](https://github.com/giantswarm/releases/pull/1343/files#diff-5a3f412a2ac88144a057a5a60cdd40d476c3cd5aa799841c1574702ba4536a75R7), but forgot to do it for the diff. Will update that. As a side note - `devctl` does not generate release names with that prefix when it seems like it should. If users are intended to specify it manually, we should update the release creation documentation to reflect that.

Yes you're right, if teams collaborate on a release, the diff is only ever done client-side